builder/build.go

```diff
@@ -231,19 +231,21 @@ func Build(pkgName, outpath string, config *compileopts.Config, action func(Buil
 					newGlobal.SetName(name)
 				}
 
-				// Try to interpret package initializers at compile time.
-				// It may only be possible to do this partially, in which case
-				// it is completed after all IR files are linked.
-				pkgInit := mod.NamedFunction(pkg.Pkg.Path() + ".init")
-				if pkgInit.IsNil() {
-					panic("init not found for " + pkg.Pkg.Path())
-				}
-				err := interp.RunFunc(pkgInit, config.DumpSSA())
-				if err != nil {
-					return err
-				}
-				if err := llvm.VerifyModule(mod, llvm.PrintMessageAction); err != nil {
-					return errors.New("verification error after interpreting " + pkgInit.Name())
+				if config.InterpPackage() {
```

Comment:
This configuration option has an effect on the cached bitcode for the package, so must be included in the `packageAction` struct. Otherwise you're going to end up with incorrect/stale caches.

Reply:
Done.